README.md

```diff
@@ -60,6 +60,7 @@ This preview version currently includes built-in support for the following Cordo
 * [cordova-plugin-file](https://github.com/apache/cordova-plugin-file)
 * [cordova-plugin-geolocation](https://github.com/apache/cordova-plugin-geolocation)
 * [cordova-plugin-globalization](https://github.com/apache/cordova-plugin-globalization)
+* [cordova-plugin-inappbrowser](https://github.com/apache/cordova-plugin-inappbrowser)
```

Comment:
Did you reuse some code from the plugin's repo? If yes, we might need to update the third-party notices